components/Organization/New/Step2DescribeOrganization.vue

```diff
@@ -259,6 +259,17 @@
             </div>
           </LinkedToAccordion>
         </fieldset>
+        <SearchableSelect 
+          v-if="isGlobalAdmin && 'badges' in organization"
```

Comment:
Why the `'badges' in organization"`?

Reply:
Because `DescribeOrganization` could be used in the creation form where it's a `NewOrganization` type without badge. Not sure we want to support adding badge during creation?